test/groovy/util/JenkinsFileExistsRule.groovy

```diff
@@ -15,6 +15,11 @@ class JenkinsFileExistsRule implements TestRule {
      */
     final List queriedFiles = []
 
+    JenkinsFileExistsRule(BasePipelineTest testInstance) {
+        this.testInstance = testInstance
```

Comment:
I would like to suggest to forward to the other already existing constructor:

```
JenkinsFileExistsRule(BasePipelineTest testInstance) {
    this(testInstance, [])
}
```
Advantage: code is better maintainable. In case there is a need for doing something else in the constructor there is no need for doing it twice (...and forgetting it once).